src/backends/glvisualize.jl

```diff
@@ -692,7 +692,7 @@ function draw_ticks(
     text, positions, offsets
 end
 
-function text(position, text, kw_args)
+function glvisualize_textposition, text, kw_args)
```

Comment:
Sorry everyone. I should have tested first - trying to cram too much into too little time. (Also I am really looking forward to those tests working again :-) )